File: LP_Python/loop_looping/lista_loop/loop_3.py
RESTO= float (0)
SOMAP= float (0)
SOMAI= float (0)
CONTP= float (0)
CONTI= float (0)
MDI= float (0)


for i in range(1,11,1):
    N = int(input("Digite %d numeros: "%i))
    RESTO = N%2
    if RESTO==0:
        CONTP+=1
        SOMAP+=N
    else:
        CONTI+=1
        SOMAI+=N
        MDI=SOMAI/CONTI

print("contagem de num pares: %d\n soma dos num pares: %d\n "%(CONTP,SOMAP))
print("contagem de num impares: %d\n media dos num impares: %.2f"%(CONTI,MDI))

File: LP_Python/loop_looping/lista_loop/loop_3.cpp
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
    float mdI;
    int n=0,somaP=0,somaI=0,countI=0,countP=0;

    for (int i = 0; i < 10; i++)
    {
        printf("Digite um número (posição= %d): \n",i+1);
        scanf("%d",&n);
        int resto=n%2;
        if (resto==0)
        {
            countP++;
            somaP+=n;
        } else {
            countI++;
            somaI+=n;
            mdI=somaI/countI;
        }

    }
        printf("Contagem de números pares: %d\n Soma dos pares: %d\n ",countP,somaP);
        printf("Contagem de números impares: %d\n Soma dos impares: %d\n Média dos impares: %.2f\n",countI,somaI,mdI);
        

    return 0;
}
